content/api/entity-reference/gift/index.html

```diff
@@ -9,379 +9,11 @@
-                <td>string</td>
-                <td>The constituent identifier.</td>
-            </tr>
-        </tbody>
-    </table>
-</div>
-
-
-<a id="GiftSplit"></a>
-<p>&nbsp;</p>
-<h2>Gift split</h2>
-<p>A gift split is the distribution of the total amount of a gift among multiple campaigns, funds, or appeals.</p>
-<p>The <b>gift split</b> entity has the following properties:</p>
-
-<div class="table-responsive">
-    <table class="table table-striped table-hover">
-        <thead>
-            <tr>
-                <th><strong>Property</strong></th>
-                <th><strong>Type</strong></th>
-                <th><strong>Description</strong></th>
-            </tr>
-        </thead>
-        <tbody>
-            <tr>
-                <td>fund_id</td>
-                <td>string</td>
-                <td>The identifier of the fund associated with the gift split.</td>
-            </tr>
-            <tr>
-                <td>amount</td>
-                <td><a href="{{ stache.config.constituent_entity_reference }}#Currency">currency</a></td>
-                <td>The amount of the gift split.</td>
-            </tr>
-            <tr>
-                <td>campaign_id</td>
-                <td>string</td>
-                <td>The identifier of the campaign associated with the gift split.</td>
-            </tr>
-            <tr>
-                <td>appeal_id</td>
-                <td>string</td>
-                <td>The identifier of the appeal associated with the gift split.</td>
-            </tr>
-        </tbody>
-    </table>
-</div>
-
-
-<a id="SoftCredit"></a>
-<p>&nbsp;</p>
-<h2>Soft credit</h2>
-<p>A soft credit is a way to share the credit for a gift with other individuals such as friends or spouses.</p>
-<p>The <b>soft credit</b> entity has the following properties:</p>
-
-<div class="table-responsive">
-    <table class="table table-striped table-hover">
-        <thead>
-            <tr>
-                <th><strong>Property</strong></th>
-                <th><strong>Type</strong></th>
-                <th><strong>Description</strong></th>
-            </tr>
-        </thead>
-        <tbody>
-            <tr>
-                <td>amount</td>
-                <td><a href="{{ stache.config.constituent_entity_reference }}#Currency">currency</a></td>
-                <td>The amount of the soft credit.</td>
-            </tr>
-            <tr>
-                <td>constituent_id</td>
-                <td>string</td>
-                <td>The constituent identifier.</td>
-            </tr>
-            <tr>
-                <td>id</td>
-                <td>string</td>
-                <td>The soft credit identifier.</td>
-            </tr>
-            <tr>
-                <td>gift_id</td>
-                <td>string</td>
-                <td>The gift identifier.</td>
-            </tr>
-        </tbody>
-    </table>
+<div ng-app="entityReferenceApp">
+  <entity-reference
+    title="Gift"
```

Comment:
I like the attributes. I think we might want to prefix them, so they don't conflict w/ HTML attributes. For example, "er-title" or "bb-title" or something...